development/src/pages/home/home.scss

```diff
@@ -89,130 +87,129 @@ h2 {
   padding-bottom: 20px;
 }
 
-.skillset {
+.skills {
 	padding: 20px 40px;
 	margin-left: -40px;
 	margin-right: -40px;
   background-color: #d5d1ce;
-}
-
-.skill-List {
-		display: grid;
+  &__listgroup{
+    display: grid;
 		grid-template-columns: repeat(auto-fit, minmax(150px, 1fr));
 		gap: 20px 30px;
-		li {
+		&__item {
 			background-color: #fff;
 			padding: 12px 10px;
 			border-radius: 0 3px 3px 0;
 			border-left: 5px solid #15717d;
 			font-size: 1rem;
 		}
-	}
+  }
+}
 
-.experienceList {
-		li.card {
-				box-shadow: 5px 9px 10px 4px #f0efed;
-				padding: 20px;
-				border-radius: 10px;
-				border: 1px solid #f1f1f1;
-				display: flex;
-        flex-direction: column;
-				line-height: 1.4rem;
-				font-size: 1rem;
-				gap: 30px;
-				margin-bottom: 20px;
+.experience{
+  &__listgroup{
+    &__item{
+      box-shadow: 5px 9px 10px 4px #f0efed;
+      padding: 20px;
+      border-radius: 10px;
+      border: 1px solid #f1f1f1;
+      display: flex;
+      flex-direction: column;
+      line-height: 1.4rem;
+      font-size: 1rem;
+      gap: 30px;
+      margin-bottom: 20px;
+      &__left{
+        line-height: 2rem;
+	      width: 220px;
+        &__tenure{
+          position: relative;
+          background: #15717d;
+          padding: 4px 15px 4px 28px;
+          color: #fff;
+          margin-left: -30px;
+          margin-bottom: 8px;
+          &:before {
+            content: '';
+            position: absolute;
+            border: 5px solid transparent;
+            border-right-color: #165f69;
+            border-top-color: #165f69;
+            bottom: -10px;
+            left: 0;
+          }
+        }
+      }
+      &__right{
+        width: 100%;
+		    border-left: 2px dotted #15717d;
+		    padding-left: 20px;
+        &__listgroup{
+          &__item{
+            margin-bottom: 8px;
+						position: relative;
+            &::before {
+              content: '';
+              position: absolute;
+              width: 12px;
+              height: 12px;
+              background-color: #fff;
+              border: 2px solid #15717d;
+              border-radius: 10px;
+              left: -27px;
+              top: 4px;
+            }
+            &::after {
+              content: '';
+              position: absolute;
+              width: 8px;
+              height: 2px;
+              background-color: #15717d;
+              left: -16px;
+              top: 9px;
+            }
+          }
+        }
         @media (min-width: 768px){
```

Comment:
@Adititomar123 i would also create a breakpoint variable, and use it across. this will avoid typo on break point, etc.

Reply:
@Adititomar123 i would also create meaningful variable names for colors

Reply:
done